MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -255,6 +255,19 @@
                 }
 				break;
 #endif
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_DRIVER_PROPERTIES: {
+				auto* physicalDeviceDriverProps = (VkPhysicalDeviceDriverPropertiesKHR*)next;
+				char mvkVersion[16], vkVersion[16];
+				vkGetVersionStringsMVK(mvkVersion, sizeof(mvkVersion), vkVersion, sizeof(vkVersion));
+				strcpy(physicalDeviceDriverProps->driverName, "MoltenVK");
+				strcpy(physicalDeviceDriverProps->driverInfo, mvkVersion);
+				physicalDeviceDriverProps->driverID = (VkDriverId) 0; /* FIXME: VK_DRIVER_ID_MOLTENVK_KHR */
```

Comment:
I've submitted a request to the Vulkan Working Group to add `VK_DRIVER_ID_MOLTENVK` to `VkDriverId`. 

Once it gets approved and released into the Vulkan headers, I'll let you know, and you can modify this accordingly, and update the value in `Vulkan-Headers_repo_revision` to use the updated Vulkan headers.

We'll hold this PR here until that happens.

Reply:
`VK_DRIVER_ID_MOLTENVK ` is available now. I'm going to pull this in and add that.